### PR TITLE
[DCOS 62643] [COPS 5709] Prometheus S3 artifacts migration and 'Virtual Network Enabled' bug fix.

### PR DESCRIPTION
**Pull request by ankitcid**
## What changes were proposed in this pull request?
Resolves [DCOS-62643](https://jira.mesosphere.com/browse/DCOS-62643) , Resolves [COPS-5709](https://jira.mesosphere.com/browse/COPS-5709)
DCOS-62643:
Migrated the S3 bucket artifacts to downloads.mesosphere.com to avoid the issues after S3 bucket deletion. Hence changing the existing resource-uris, images paths.
COPS-5709:
Created a new version 0.1.2-2.3.2 regarding the Permanent fix for COPS-5709. Service will run with virtual network enabled.

## How were these changes tested?
- Successfully deployed the Prometheus service on DCOS 2.0 cluster.
- Soaked versions 0.1.0-2.3.2, 0.1.1-2.3.2, 0.1.2-2.3.2 on soak200s, soak201s soak clusters.

## Release Notes
User can use the service with virtual network enabled.

### CODE REVIEW

**Comment by ankitcid** [comment]
Updated **DCOS-62643-Prometheus-Migrate-artifacts-to-downloads.mesosphere.com** branch with soaked changes.

File: repo/packages/P/prometheus/2/resource.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                "pushgateway": "prom/pushgateway:v0.5.2"
+            }
+        }, 
+        "uris": {
+            "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz", 
+            "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.11.0.tar.gz", 
+            "bootstrap-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.42.1/bootstrap.zip", 
+            "executor-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.42.1/executor.zip", 
+            "scheduler-zip": "https://downloads.mesosphere.io/dcosdev-sdk/assets/0.42.1/operator-scheduler.zip", 
+            "svc": "https://arathorec-test.s3.amazonaws.com/prometheus/artifacts/0.1.2-2.3.2/svc.yml",
```
Comment (realmbgl) [inline repo/packages/P/prometheus/2/resource.json]: hi @ankitcid you will have to change the URI for the svc.yml to the downloads one.

Comment (ankitcid) [reply repo/packages/P/prometheus/2/resource.json]: Hi @realmbgl , done the change in 0.1.2-2.3.2 version.

**Review by rishabh96b — APPROVED** [review]
LGTM :heavy_check_mark:

**Review by vibhujain — APPROVED** [review]
LGTM :+1: . Thanks Ankit.

**Comment by ankitcid** [comment]
Merging the PR after approvals and passing all CI Checks.

**Pull request merged**